root/data-emacs.yml

```diff
@@ -39,16 +39,16 @@
         help: Aller à la dernière ligne de l'historique
       - cmds:
           - Ctrl-r
-        help: Rechercher antérieurement dans l'historique
+        help: Rechercher en arrière dans l'historique
       - cmds:
           - Ctrl-s
-        help: Rechercher postérieurement dans l'historique
+        help: Rechercher en avant dans l'historique
```

Comment:
Same idea than https://github.com/savoirfairelinux/readline-cheat-sheet/pull/1/files#r109235534

What do you think of:

Rechercher **en ordre chronologique** dans l'historique